Converter.cpp

```diff
@@ -6,8 +6,8 @@ Returns the next frame if next frame is recorded
 Returns the previous frame if next frame is not recorded
 ***/
 void Converter::ConvertPXCImageToOpenCVMat(Intel::RealSense::Image *inImg, Intel::RealSense::ImageData data, cv::Mat *outImg) {
-	auto cvDataType = 0;
```

Comment:
Sorry there are some totally irrelevant changes, I was looking around to find the source of the memory leak. undo this if needed.